EIPS/eip-7727.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 7727
+title: EVM Transaction Bundles
+description: Enable EVM support for transaction bundles without revert protections.
+author: Lily Johnson (@lilyjjo)
+discussions-to: https://ethereum-magicians.org/t/eip-7727-evm-transaction-bundles/20322
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-06-24
+requires: 2718
+---
+
+## Abstract
+
+This EIP introduces two new [EIP-2718](./eip-2718.md) transaction types and one new opcode, enabling smart contracts and transactions to delegate their local sequencing rights to an off-chain entity. These new transaction types work together to create EVM-native 'bundles', which are similar but weaker than the PBS bundles offered by builders to searchers.
+
+One of the EIP-2718 transactions is an extended normal transaction that supports:
+
+- Specifying the entity who can place the transaction in a bundle.
+- An optional block number that the transaction is valid in.
+
+The other EIP-2718 transaction is a ‘meta’ transaction that does not enter into execution, but rather orders transactions of its own type and the other new type. The 'meta' transaction can also specify an entity allowed to include it in a bundle and a valid block number.
+
+The new opcode reveals to the EVM the entity that placed the transaction in a bundle if the transaction was in a bundle.
```

Comment:
This is the correct level of detail for the abstract, but it's a bit long. If possible, please condense it down a bit.